tests/test_unit/test_helpers.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) 2018, Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test expected functioning of the helper functions."""
+
+
+import cobra
+import pytest
+from cameo.strain_design.pathway_prediction.pathway_predictor import (
+    PathwayResult)
+
+import metabolic_ninja.helpers as helpers
+
+
+model_registry = {}
+pathway_registry = {}
+
+
+def register_with(registry):
```

Comment:
This registry pattern looks a bit odd to me, how does this work? I suppose it's got something to do with the `indirect` argument to `pytest.parametrize`?